src/main/java/org/sigmah/shared/dto/referential/ProjectModelStatus.java

```diff
@@ -133,6 +133,10 @@ public static ProjectModelStatus getStatus(final String status) {
 		} else if (currentStatus.equals(ProjectModelStatus.USED) && !targetStatus.equals(ProjectModelStatus.UNAVAILABLE)) {
 			// "USED" status is only allowed to shift to "UNAVAILABLE" status.
 			return new Pair<Boolean, String>(false, I18N.CONSTANTS.usedModelStatusChangeError());
+			
+		}else if (currentStatus == ProjectModelStatus.UNAVAILABLE && targetStatus == ProjectModelStatus.USED) {
+			// for #199: When "Unavailable" a project model can not be "Available" again

```

Comment:
Commenting is always a good practice !
And more comment is better that not enough. But we have a (newly explicited...) rule :  [**No reference to issue in source code comment** ](http://wiki.sigmah.org/doku.php?id=contributorguide:contributionrules#make_your_work_easy_to_merge). So this comment should rather be replaced by something